AndelaOSP/wire
# fix change of assignee in sidebar clearing reporter information

#### What does this PR do?

Fixes the bug that makes the reporter's name clear off once an assignee has been selected.
#### Description of Task to be completed?

As an admin, I should be able to assign a user to an incident, currently, this clears out the name of the reporter on the timeline.

#### Where should the reviewer start?
N/A
#### How should this be manually tested?
* Clone this project.
* Checkout `bg-persist-reporter-record-on-ui`
* Log into the dashboard as an admin
* Select a single incident to view the incident details.
* Change the assignee, then confirm that the reporter's details doesn't change.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
